### PR TITLE
Tag VegaLite.jl v0.0.2

### PR DESCRIPTION
**Pull request by attobot**
Repository: [fredo-dedup/VegaLite.jl](https://github.com/fredo-dedup/VegaLite.jl)
Release: [v0.0.2](https://github.com/fredo-dedup/VegaLite.jl/releases/tag/v0.0.2)
Travis: [Travis Build Status](https://travis-ci.org/fredo-dedup/VegaLite.jl/builds/208360746)
Diff: [vs v0.0.1](https://github.com/fredo-dedup/VegaLite.jl/compare/a63916c506b1f70d13a96a43dd0455ab25d0df2c...c46bc8de360792893d58a6800758b19da41d5e04)
`requires` vs v0.0.1: 
```diff
--- 0.0.1/requires
+++ 0.0.2/requires
@@ -1,3 +1,4 @@
 julia 0.4
 Compat
 JSON
+Requires
```
cc: @fredo-dedup

### CODE REVIEW

**Comment by ararslan** [comment]
`Pkg.installed` isn't a good way to do optional dependencies in packages, since it doesn't play well with precompilation. Since you added a dependency on Requires.jl, you could just use that.

**Comment by tkelman** [comment]
Should also really mention the licenses of all bundled code in your top-level license file.

File: VegaLite/versions/0.0.2/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.4
+Compat
```
Comment (tkelman) [inline VegaLite/versions/0.0.2/requires]: It's best to declare a minimum version dependency that implements the features you need.

**Comment by fredo-dedup** [comment]
Thank you for your advices. 
Updated license file + removed Pkg.installed() calls + added a version to compat

**Pull request merged**